src/libraries/Directory.Build.targets

```diff
@@ -295,4 +296,24 @@
     <DefaultValidateFramework Remove="net6.0" />
   </ItemGroup>
 
+  <PropertyGroup Condition="Exists('$(PkgProjPath)')"> 
+    <ServicingVersion Condition="'$(ServicingVersion)' == ''">0</ServicingVersion>
+    <!-- Always update the package version.-->
+    <PackageVersion>$(MajorVersion).$(MinorVersion).$(ServicingVersion)</PackageVersion>
```

Comment:
I do see that Arcade defines `PatchVersion` would that be appropriate to use here?
https://github.com/dotnet/arcade/blob/6cc4c1e9e23d5e65e88a8a57216b3d91e9b3d8db/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L76

Reply:
We wont be able to use it for assembly version as its incremented with every release whereas assembly last digit is pinned to 0

Reply:
I see. Glad we have a good reason for being different.